Vagrantfile

```diff
@@ -11,7 +11,7 @@ CONFIG = File.join(File.dirname(__FILE__), "config.rb")
 # Defaults for config options defined in CONFIG
 $num_instances = 1
 $instance_name_prefix = "core"
-$update_channel = "alpha"
```

Comment:
I guess I was thinking from our product's point of view, where we have nightly jobs that test against different update channels, and just prefixing the command is simple: `COREOS_CHANNEL=alpha ./tests/bin/test-integration.sh`. But `config.rb` works, and I didn't realize there weren't any other env vars in use.